api-reference/pagination.mdx

```diff
@@ -3,48 +3,6 @@ title: "Pagination"
 description: ""
 ---
 
-*Placeholder below*
+Some top-level API resources have support for bulk fetches through "list" API methods. You may list API keys, list plans, list companies, list users, list events, list features, list flags, and list entitlements. These list API methods share a common structure and accept the following parameters: `limit` and `offset`. Depending on the resource, they also accept additional filtering parameters.
```

Comment:
listing resources here seems like a lot to maintain - we won't remember to update this sentence every time we add a list endpoint

Reply:
good call - will revise.